### DISCUSSION TITLE
None of the validators work on the table repeater fields

### DISCUSSION THREAD

**Discussion by ecoy1** [awcodes/filament-table-repeater#134] [Q&A]
Hi,

None of the rules I try to add on the fields seem to work. I have table repeaters in an Edit page and I've tried using ->required() or ->nullable(false) on the fields yet none of them prevent the form submission or give errors at the front. I have also used ->markAsRequired() but this only adds the * to the header. 

Here's an image of the desired functionality I am looking for (this is with using the built-in Repeater). How can I achieve this? If noti what is the alternative?

<img width="548" alt="image" src="https://github.com/user-attachments/assets/da8d01f2-91d4-410a-9710-425683c9b5ee">

Thank you!
Ece

**Comment by awcodes** [comment]
Validations are working correctly in my demo / testing app.

![Screenshot 2024-12-03 at 12 33 13 PM](https://github.com/user-attachments/assets/051cbf97-ddd3-43fb-99bc-039066b41488)
